content/docs/04.5-devx/07-sandbox-clusters.md

```diff
@@ -47,3 +47,42 @@ To create your new sandbox cluster:
    * Review the information and deploy the Palette virtual cluster. The Palette virtual cluster will be provisioned within the next few minutes.
 
 <br />
+
+
+## Palette Virtual Cluster Pause and Resume
+
+Palette allows the pause and resume of Palette Virtual Clusters when not in use. This feature enables the users to optimize resource utilization by pausing the Virtual Clusters, not in use. This adds significant flexibility in managing operating costs and resource management for the Palette Virtual Clusters. 
+
+<br />
+
+### System and Resource Impact
+
+* The quota allocation is independent of a Virtual Cluster's Pause or Resume status.
+
+
+* The CPU and Memory are freed to the Cluster Group upon pausing a Virtual Cluster.
+
+
+* Resources such as storage, and Load Balancers remain allocated to a Virtual Cluster regardless of the state.
+
+
+* The Apps deployed on a Virtual Cluster go to a pause state when the cluster is paused.
+
+
+* New Apps cannot be deployed on a Virtual Cluster in the paused state.
+
+
+* Virtual Clusters in a paused state will continue to appear as an entry in the Palette Dev Engine Console. 
+
+<br />
+
```

Comment:
This is great, but we need to explain to the user what the link below is for. It can be as short as a sentence but let's always provide context to the reader. As an example -  "To learn how to pause and resume your Palette Virtual cluster. Check out the how-to guide."